ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -60,7 +60,7 @@ images:
     openshift_release_golang-1.18:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.18
-  to: cluster-api-kubevirt-controller
+  to: cluster-api-provider-kubevirt
```

Comment:
What is the reason to change this?
This was the same convention as other platforms

Reply:
@nirarg  not all providers, for example agent [1]  follows the `cluster-api-provider-*` convention. The `cluster-api-provider-*` convention is also what is used downstream in the mce builds. I think aligning the repo name, project name, and container image name will be more consistent for us. It's possible some of these other providers used that different convention to avoid colliding with the old machine controllers by the same name.

I left the release-1.0 branch with the old convention in order not to break the existing hypershift CI. I plan to move us away from that release-1.0 branch shortly after this change goes in.


1. https://github.com/openshift/release/blob/6e38a4cd8dc10106eb4f849885b0376f517c9abd/ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml#L47

Reply:
all our mirrored names are of the form $cloudprovider-cluster-api-controllers

$ grep cluster-api mapping_origin_4_11 | awk '{print $3}'
quay.io/openshift/origin-aws-cluster-api-controllers:4.11.0
quay.io/openshift/origin-azure-cluster-api-controllers:4.11.0
quay.io/openshift/origin-gcp-cluster-api-controllers:4.11.0
quay.io/openshift/origin-vsphere-cluster-api-controllers:4.11.0

the exceptions are these, one of which is kubevirt and the other 2 are not provider specific:

quay.io/openshift/origin-cluster-api-actuator-pkg:4.11.0
quay.io/openshift/origin-cluster-kube-cluster-api-operator:4.11.0
quay.io/openshift/origin-cluster-api-kubevirt-controller:4.11.0

The expected conventions for image mirroring names are here:
https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md

So if anything, we should be renaming the kubevirt one to "origin-kubevirt-cluster-api-controllers" to be consistent.

i'd be amenable to "origin-kubevirt-cluster-api-provider", but the cloudprovider should be the first segment.

Reply:
@bparees good point, I've updated the images to be consistent with the other capi providers. `kubevirt-cluster-api-controllers`